src/components/App/ExpressRecorder.tsx

```diff
@@ -481,6 +481,7 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
             });
         }
         // update constraints state only if create stream has been succeeded
+        this.stopStreams(screenOn);
         this.createStream(newConstraints, screenOn, toggleWasChanged);
```

Comment:
warning with this approach. you will lose control of execution time.
if those commands should run in parallel, so I'm fine with that.
but, if they relay on each other, so use a callback function or Promise resolve.
```suggestion
        this.stopStreams(screenOn, () => this.createStream(newConstraints, screenOn, toggleWasChanged));
```

Reply:
stop streams is not asyncronous code.. so it's fine